src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/ITopicProducerProvider.cs

```diff
@@ -1,4 +1,4 @@
-namespace MassTransit.KafkaIntegration
+namespace MassTransit
```

Comment:
Should move to another folder so that namespace matches folder.

Reply:
The root namespace of every project is MassTransit. That's why we have the folder structure in each project.